llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -1294,6 +1294,26 @@ static bool isLoopPassName(StringRef Name, CallbacksT &Callbacks,
   return callbacksAcceptPassName<LoopPassManager>(Name, Callbacks);
 }
 
+static StringRef convertToNewPassName(StringRef OldName) {
+  static StringMap<StringRef> NameMap = {{"callbrprepare", "callbr-prepare"},
+                                         {"dwarfehprepare", "dwarf-eh-prepare"},
+                                         {"flattencfg", "flatten-cfg"},
+                                         {"loweratomic", "lower-atomic"},
+                                         {"lowerinvoke", "lower-invoke"},
+                                         {"lowerswitch", "lower-switch"},
+                                         {"winehprepare", "win-eh-prepare"},
+                                         {"targetir", "target-ir"},
+                                         {"targetlibinfo", "target-lib-info"}};
```

Comment:
This set specifically I don't think will be disruptive enough to justify maintaining aliases

Reply:
I'd still drop this alias handling, especially given there's no additional testing for these names